app/src/main/java/de/tum/in/tumcampusapp/component/tumui/roomfinder/NavigationDetailsFragment.kt

```diff
@@ -89,13 +89,29 @@ class NavigationDetailsFragment : BaseFragment<Unit>(
         binding.locationName.text = navigationDetails.name
         binding.locationType.text = getCapitalizeType(navigationDetails.typeCommonName)
 
+        setupShareLocationButton(navigationDetails)
         setOpenInOtherAppBtnListener(navigationDetails)
 
         showNavigationDetailsProperties(navigationDetails)
 
         showAvailableMaps(navigationDetails)
     }
 
+    private fun setupShareLocationButton(navigationDetails: NavigationDetails) {
+        val url = "https://nav.tum.sexy/${navigationDetails.type}/${navigationDetails.id}"
```

Comment:
This is sadly not 100% correct. Either we do this:
```suggestion
        val url = "https://nav.tum.sexy/view/${navigationDetails.id}"
```

Or steal this code:
https://github.com/TUM-Dev/NavigaTUM/blob/main/webclient/src/views/view/view-view.js#L24-L35